project/deps.sc

```diff
@@ -169,8 +169,13 @@ object Versions {
   val `zio-schema`    = "0.4.12"
 }
 
+object DevMode {
```

Comment:
The build kind of already has a means of getting in user specific properties, and it uses a file in order to be incremental build friendly:

https://github.com/finos/morphir-scala/blob/main/mill-build/src/millbuild/CommonCrossScalaModule.scala#L37